corona-warn-app/cwa-app-android
# DCC Ticketing Consent I (EXPOSUREAPP-10471)

## Whats new
DCC Ticketing Consent I screen

![Screenshot 2021-11-16 at 22 08 36](https://user-images.githubusercontent.com/64849422/142050115-81d0d296-1720-4490-b949-f495a9fe32ad.png)


## Ticket
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10471)
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194)

## How to test
1) Open Dev Menu
2) Scroll down to DCC Ticketing
3) Press "Open Consent I Screen" button
4) Check the consent screen

## ToDo
- [x] View model for this screen?
